backend/btrixcloud/subs.py

```diff
@@ -395,6 +438,17 @@ async def cancel_subscription(
     ):
         return await ops.cancel_subscription(cancel)
 
+    @app.post(
+        "/subscriptions/trial-end-reminder",
```

Comment:
```suggestion
        "/subscriptions/send-trial-end-reminder",
```
Maybe should rename this for clarity that all this does is send the reminder?